### ISSUE TITLE
input file checking for blastparser

### ISSUE THREAD

**Issue by mvdbeek** [ARTbio/tools-artbio#65]
@drosofff I saw [this](https://mississippi.snv.jussieu.fr/reports/jobs/specified_date_handler?sort=id&f-specified_date=2016-03-14&webapp=reports&operation=job_info&id=639d1948d5e1fcb4) in the reports this morning:

```
Traceback (most recent call last):
  File "/home/galaxy/shed_tools/toolshed.g2.bx.psu.edu/repos/drosofff/msp_blastparser_and_hits/77346ecf5a28/msp_blastparser_and_hits/BlastParser_and_hits.py", line 231, in 
    if __name__=="__main__": __main__()
  File "/home/galaxy/shed_tools/toolshed.g2.bx.psu.edu/repos/drosofff/msp_blastparser_and_hits/77346ecf5a28/msp_blastparser_and_hits/BlastParser_and_hits.py", line 221, in __main__
    Xblastdict = getblast (args.blast)
  File "/home/galaxy/shed_tools/toolshed.g2.bx.psu.edu/repos/drosofff/msp_blastparser_and_hits/77346ecf5a28/msp_blastparser_and_hits/BlastParser_and_hits.py", line 85, in getblast
    blastinfo.append(int(fields[12])) # blastinfo[10] Subject length MUST BE RETRIEVED THROUGH A 13 COLUMN BLAST OUTPUT
ValueError: invalid literal for int() with base 10: 'gi|507434208|dbj|HW087993.1|_WO_2013027427-A/49:_Conditional'
```

I guess it's an input file problem, perhaps we can check that the input file has 13 columns in the tool wrapper, and catch ValueError in the code and raise an error with an easier to understand message.

**Comment by drosofff** [comment]
Yes totally agree. 

Christophe Antoniewski
Drosophila Genetics and Epigenetics
Laboratoire de Biolologie du Développement
9, Quai St Bernard, Boîte courrier 24
75252 Paris Cedex 05

Tel		+33 1 44 27 34 39
Fax		+33 1 44 27 34 45
Mobile	+33 6 68 60 51 50

http://drosophile.org

**Comment by JuPeg** [comment]
Also, some information about the format expected could be added in the "What it does". Specially since the 13 columns format is not a standard blast output format.

**Comment by drosofff** [comment]
The help section is now much more explicit. Still have to capture errors du to tabular blast output with missing 13th column.

**Comment by drosofff** [comment]
@mvdbeek wrote:

> I guess it's an input file problem, perhaps we can check that the input file has 13 columns in the tool wrapper, and catch ValueError in the code and raise an error with an easier to understand message.

@ccedmendoza it's an easy fix. Can we work it at some point ?

**Comment by ccedmendoza** [comment]
Ok I will look for a way to do this

**Comment by drosofff** [comment]
Ping @ccedmendoza

**Comment by drosofff** [comment]
solved by PR #223
